content/en/security/application_security/enabling/compatibility/java.md

```diff
@@ -6,45 +6,131 @@ type: multi-code-lang
 code_lang_weight: 0
 ---
 
+## ASM capabilities support
+
+The following ASM capabilities are supported in the Java library, for the specified tracer version:
+
+| ASM capability                   | Minimum Java tracer version |
+| -------------------------------- | ----------------------------|
+| Threat Detection | 1.8.0  |
+| Threat Protection| 1.9.0 |
+| Open Source Vulnerability detection | 1.1.4 |
+| Custom Code Vulnerability detection | 1.15.0|
+
+The minimum tracer version to get all supported ASM capabilities for Java is 1.15.0.
+
+**Note**: Threat Protection requires enabling [Remote Configuration][2], which is included in the listed minimum tracer version.      
+
 ## Language and framework compatibility
 
 ### Supported Java versions
+The Java Tracer supports automatic instrumentation for the following Oracle JDK and OpenJDK JVM runtimes.
 
-The Datadog library supports Java JRE 1.8 and higher of both Oracle JDK and OpenJDK, on the following architectures:
-- Linux (GNU) x86, x86-64
-- Alpine Linux (musl) x86, x86-64
-- macOS (Darwin) x86, x86-64
-- Windows (msvc) x86, x86-64
+| JVM versions | Operating Systems                                                               | Support level                       | Tracer version |
+| -------------| ------------------------------------------------------------------------------- | ----------------------------------- | -------------- |
+| 18 to 19     | Windows (x86, x86-64)<br>Linux (x86, x86-64, arm64)<br>Mac (x86, x86-64, arm64) | [Beta](#levels-of-support)               | Latest         |
```

Comment:
@sourabhkatti These links here for "Beta" , GA, End-of-life etc don't go anywhere, should there be a description table somewhere or should those links be removed?

Reply:
Ah actually this is wrong. We should only have two rows: Supported and Not Supported. The links should be deleted. I'll update this table